torch/distributed/distributed_c10d.py

```diff
@@ -218,6 +218,26 @@ def supports_complex(reduceOp: ReduceOp) -> bool:
     return reduceOp not in denyList
 
 
+def get_backend_from_device(device) -> str:
```

Comment:
hm, now that I see this, I realize that there is not one backend per device.  "cuda" can be used with mpi, for instance.

i think this helper is still useful, perhaps we should rename it to 'get_default_backend'?

and it looks like we have an incomplete 'default_device_backend_map' inside Backend class. lets update that and make this helper return from that instead?

In fact, maybe this should just be a staticmethod inside Backend class?

Reply:
thanks for the feedback.
I have moved the function inside Backend class as static method.
I think there is no need to update the 'default_device_backend_map' inside class as for new backend this map gets updated with existing functionality of register_backend() API.

Please review.